common/theme.js

```diff
@@ -3,21 +3,22 @@ import PropTypes, { number, exact, string, oneOfType } from 'prop-types';
 import { ThemeContext } from 'styled-components';
 
 /* eslint-disable */
-const defaultTheme = require('sass-extract-loader?{"plugins": ["sass-extract-js"]}!../styles/default/_theme-variables.scss');
+const defaultTheme = require('styles/default/theme.json');
+
 require('../styles/default/main.scss');
 
 const themeCache = {};
 
 if (!process.browser) {
-  // Import the _theme-variables.scss files from all the themes and store them
+  // Import the theme.json files from all the themes and store them
```

Comment:
@juyrjola I guess we still cache the themes in the same way? Just load them as json directly.